src/main/java/nl/tudelft/cse1110/andy/writer/weblab/WebLabResultWriter.java

```diff
@@ -15,8 +16,8 @@
 
 public class WebLabResultWriter extends StandardResultWriter {
 
-    public WebLabResultWriter(RandomAsciiArtGenerator asciiArtGenerator) {
-        super(asciiArtGenerator);
+    public WebLabResultWriter(VersionInformation versionInformation, RandomAsciiArtGenerator asciiArtGenerator) {
```

Comment:
Unless needed for tests, I would not create this constructor

Reply:
Ok, it's needed for the tests. So I'd create an extra constructor that only receives ascii generator, and that's the one clients will use.

Maybe even make this one that receives the property to be with default visibility, so that only the test can use it

Reply:
I don't think we can do that - the test is in a different package.

Reply:
True, we can leave it public.